_scripts/instruction-widget/install.js

```diff
@@ -251,6 +254,14 @@ module.exports = function(context) {
     context.dns_package_prefix = "python-certbot-dns";
   }
 
+  snap_install = function() {
+    template = "snap";
+    context.base_command = "certbot";
+    context.cron_included = true;
+    context.install_command = "sudo snap install --beta --classic";
+    context.package = "certbot";
```

Comment:
just to check, we don't have separate packages for apache or nginx, they're all in there?

Reply:
Yep!